packages/docs/docs/concepts/cornerstone-core/viewportReferencePresentation.md

```diff
@@ -0,0 +1,228 @@
+---
+id: viewportReferencePresentation
+title: Viewport Image Selection Reference and Presentation
+---
+
+# Viewport Image Selection Reference and Presentation
+
+The reference and presentation information for a viewport specify what image a viewport
+is displaying, and the presentation of the image. These are specified in several ways
+so that a view can be transfered from one viewport to another, or can be remembered in order
+to restore a view later. Getting a reference can be done either for the current image,
+or a specific image in the stack, ordered/numbered in the same way that the scrolling positions
+are numbered/ordered.
+
+Some specific use cases for this are:
+
+- Referencing a specific image for a tool
+  - Uses `ViewReference` to specify an image to apply to
+  - Uses `isReferenceCompatible` to determine if the tool should be displayed or not
+  - Uses `isReferenceCompatible` to determine which of a set of viewports is best suited to navigating to an image
+  - Uses `setView(viewRef)` to navigate to the specified image
+- Restoring an earlier view or converting from stack to volume or vice-versa
+  - Uses `ViewReference` and `ViewPresentation` to store the image information
+- Applying interpolation to image sets
+  - Uses `getViewReference` with a specific image position to get references to
+    images in between or related to nearby annotations to interpolate
+- Resizing and sync display presentation
+  - Uses `getViewPresentation` to get old presentation information and then
+    restores with `setView(null,viewPres)`
```

Comment:
Can we change `setView` to an object-based approach, like `setView({})`? It seems too early, and I don't think people use it at all. So, can we address this issue of having to pass `null` as a parameter?

Reply:
Changed to setViewReference and setViewPresentation as agreed.